src/components/segmentedControl/index.tsx

```diff
@@ -164,6 +169,7 @@ const SegmentedControl = (props: SegmentedControlProps) => {
           selectedIndex={animatedSelectedIndex}
           activeColor={activeColor}
           inactiveColor={inactiveColor}
+          segmentStyle={segmentStyle}
```

Comment:
If it's an individual style for each segment, why it's not passed in the segment, meaning part of the `SegmentedControlItemProps` and not the `SegmentedControlProps`? It's confusing, or maybe its just the naming...

Reply:
Maybe it is confusing because I actually wanted one style prop for all of them. 😅 
Maybe `segmentsStyle` in plural?

Reply:
Yes, I guess `segmentsStyle` is better...

Reply:
Done :)